src/components/Header/HeaderElements/HeaderRight.jsx

```diff
@@ -35,10 +34,10 @@ export const HeaderRight = (props) => {
         <Dropdown.Menu>
           <Dropdown.Item className={styles.dropdownItem}>
             {props.isAuth
```

Comment:
never use an **inline** conditional on multiple lines,

true ? "yes" : "no" -> this is an inline conditional, if you still want to use it, you need to move your "yes" and "no" to separate constants above return.

so you can use them like:
```
{ isAuth ? LoginButton : LogoutButton }
```